docs/FRSKY_R9.md

```diff
@@ -31,7 +31,7 @@ The R9M transmitter module is a great option given its 1 W transmit power but is
 
 <img src="images/Frsky_R9M_Dip.png" width="720px">
 
-2. Dealing with the inverted TTL signals is best addressed by using a seperate ESP32 module connected to the serial port as the ESP32 supports inverted TTL signals. The mlrs-wireless-bridge sketch will allow one to connect a Ground Control Station wirelessly via MAVLink. Additionally, the CLI can be accessed with common terminal programs when using the mlrs-wireless-bridge sketch in Bluetooth Classic mode.  For ESP32 modules that do not support Bluetooth Classic (C3, C6 and S3 variants) one can use telnet or a TCP terminal program with TCP mode.
+2. Dealing with the inverted TTL signals is best addressed by using a seperate ESP32 module connected to the serial port as the ESP32 supports inverted TTL signals. The mlrs-wireless-bridge sketch will allow one to connect a Ground Control Station wirelessly via MAVLink. Additionally, the CLI can be accessed with common terminal programs when using the mlrs-wireless-bridge sketch in Bluetooth Classic mode.  For ESP32 modules that do not support Bluetooth Classic (C3, C6 and S3 variants) one can use telnet or a TCP terminal program with TCP mode.  If you use telnet, you will probably need to enable crmod mode which you can do by using the escape sequence, usually control-], and issue the "toggle crmod" command.
```

Comment:
This last sentence gave and gives me headaches. If you're not familiar with it (which I wasn't then I read it first) it comes across like gibberish and it totally unclear w.r. to what it tries to say.
My suggestion would be to add a section to the CLI documentation of how to connect to the CLI, and to reference here.
If you guys want to keep it here, I would at minimum do two things:
- mention the different OSes. On Window I just could not find any "TCP terminal program" yet which looked to me like the thing I need here. Telnet is easily available once activated in the cmd console. However here the escape sequence is CTRL++ and not CNTL-J.
- telling why this is wanted, i.e., to adjust the crlf to what the CLI uses per default

Reply:
The escape sequence is the same for me on Windows and Linux,  Perhaps this is a regional keyboard difference.  I'll mention to use the escape sequence indicated when the program starts.  I can try to clarify the rest of the sentence, but I do prefer to include these hints here at least until there is a new section to reference.

Reply:
I hope the new wording in my latest commit helps.